### DISCUSSION TITLE
Problema attribuzione rappresentate di ogni tipo

### DISCUSSION THREAD

**Discussion by tommasiangelo01** [iisgiua/giuaschool#471] [Richieste di aiuto] [answered]
Buonaseraio uso l'hosting Altervista,e ,non appena effettuo l'accesso come amministratore e mi reco nell'area "rappresentanti alunni,genitori,ata" , per attribuire appunto il ruolo,quando devo scegliere lo studente,docente,ata nel menù a tendina e come se questo fosse bloccato e non mi si apre proprio il menù a tendina

**Comment by trinko** [comment]
Stai usando la versione 1.5.2?  o hai installato anche le modifiche successive al rilascio di questa versione?

**Comment by trinko** [comment, marked as answer]
Se stai usando la 1.5.2 senza aggiornamenti successivi, leggi:

[#459](https://github.com/iisgiua/giuaschool/issues/459#issuecomment-1408189329)

**Comment by sempliciniluigi89** [comment]
Sto usando la versione 1.5.2 che ho installato sin da subito riportando il
nuovo aggiornamento da voi proposto ,senza installare versioni precedenti a
questa .



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
02/28/23,
09:55:54 AM

Il giorno mar 28 feb 2023 alle ore 00:14 Antonello Dessì <
***@***.***> ha scritto:

**Comment by sempliciniluigi89** [comment]
Ho fatto,ma purtroppo non si è risolto il problema



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
02/28/23,
09:57:02 AM

Il giorno mar 28 feb 2023 alle ore 00:17 Antonello Dessì <
***@***.***> ha scritto:

**Comment by trinko** [comment]
Hai eseguito anche il comando SQL seguente?

```
UPDATE gs_utente SET rappresentante='' WHERE rappresentante IS NULL OR rappresentante='a:0:{}';
```

**Comment by sempliciniluigi89** [comment]
si



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
02/28/23,
09:25:28 PM

Il giorno mar 28 feb 2023 alle ore 19:36 Antonello Dessì <
***@***.***> ha scritto:
